demo/src/js/components/add-remove-validation.js

```diff
@@ -0,0 +1,34 @@
+import formValidation from '@degjs/form-validation';
+import required from '@degjs/form-validation-required';
+import email from '@degjs/form-validation-email';
+import formFieldAppender from '@degjs/form-field-appender';
```

Comment:
Since you're importing from node_modules, is there any concern that the installed NPM modules will not be the latest versions? The demo is built before the modules are published to NPM, right?